public/static/docs/command-reference/checkout.md

```diff
@@ -203,23 +204,25 @@ once to download missing data from the remote storage to the <abbr>cache</abbr>.
 Alternatively, we could have just run `dvc pull` in this case to automatically
 do `dvc fetch` + `dvc checkout`.
 
-## Automating `dvc checkout`
+## Example: Automating `dvc checkout`
 
 We want the data files or directories (managed by DVC) to match with the other
 files (managed by Git e.g. source code). This requires us to remember running
 `dvc checkout` when needed, and of course we won't always remember to do so.
 Wouldn't it be nice to automate this?
 
-Let's try this:
-
 ```dvc
 $ dvc install
 ```
 
 `dvc install` installs Git hooks to automate common operations, including
 running `dvc checkout` when needed.
 
-We can then checkout the master branch again:
+> Note that the hooks only have an effect in repository versions (branches,
+> etc.) where DVC is active.
```

Comment:
@shcheklein here's that note about the hooks (per https://github.com/iterative/dvc.org/issues/506#issuecomment-576053513).
BTW, I noticed `commit` and `push` commands don't have similar references to `dvc install`, should they? Maybe not a full blown example like in here but at least a note.

Reply:
I decided to refine the existing notes and include it also in the other commands where it was missing (commit, repro, push).